core/src/test/java/org/apache/accumulo/core/conf/DeprecatedPropertyUtilTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.conf;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertThrows;
+
+import org.apache.commons.configuration2.CompositeConfiguration;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DeprecatedPropertyUtilTest {
```

Comment:
I was torn about that since the master to manager prop renaming is deprecated, and would therefore eventually need to be removed from this test. I was thinking what I added was sufficient to test the general functionality of the class, but am happy to add master to manager rename checks as well if you think it's worth it even though they'll be removed eventually.

Reply:
> I was torn about that since the master to manager prop renaming is deprecated, and would therefore eventually need to be removed from this test.

That makes sense.  It would be nice to test what the code that users will run.  When its removed in the future, if the test fails cleanly in such a way that the cause is obvious, I think it will be easy for someone to clean up the test.

Reply:
Ok, added.